cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -263,13 +266,10 @@ def upload_rna_coverage_bigwig_to_scout(self, case_id: str, dry_run: bool) -> No
                 )
 
             LOG.info(f"RNA coverage bigwig file {rna_coverage_bigwig.path} found")
-            dna_sample_id: str
-            dna_cases: List[str]
-            dna_sample_id, dna_cases = rna_dna_sample_case_map[rna_sample_id].popitem()
-            for dna_case_id in dna_cases:
+            for dna_case_id in rna_dna_bundle.dna_case_ids:
                 if self.status_db.get_case_by_internal_id(internal_id=dna_case_id).is_uploaded:
                     LOG.info(
-                        f"Uploading RNA coverage bigwig file for {dna_sample_id} in case {dna_case_id} in scout"
+                        f"Uploading RNA coverage bigwig file for {dna_sample_name} in case {dna_case_id} in scout"
```

Comment:
This info log could be improved "bigwig file for sample {dna_sample_name} in case..."